src/App.vue

```diff
@@ -1817,8 +1817,24 @@ tbody:last-child .empty-line:last-child {
   padding: 0;
 }
 
-.content ul {
-  margin: 0;
+.content {
+
+  .content ul {
```

Comment:
`.content .content` is for nested "content" elements.
I'm not sure this case exists in the Comment component.